.github/workflows/ci.yml

```diff
@@ -99,7 +99,7 @@ jobs:
       - name: Setup Python
         uses: actions/setup-python@v4
         with:
-          python-version: 3.9
+          python-version: '3.10'
```

Comment:
why quotes?

Reply:
without quotes we get following error - 

![image](https://user-images.githubusercontent.com/106588300/220538230-ddbf5e7b-ff56-4f61-b00a-9fef49be419c.png)

See here for the explanation https://github.com/actions/setup-python/issues/160.